### DISCUSSION TITLE
new issue found

### DISCUSSION THREAD

**Discussion by Ikagorgon33** [SebiAi/custom-nothing-glyph-tools#27] [Q&A] [answered]
see screen shot .
this happens when u try to do glyph translator step
![Screenshot 2024-04-03 042931](https://github.com/SebiAi/custom-nothing-glyph-tools/assets/165844673/9b5cf1e1-6b70-4603-ac22-8408a3b421df)
help what to do @SebiAi

**Comment by SebiAi** [comment, marked as answer]
TLDR: LIN/EXP/LOG not capitalized + wrong codec
